scripts/up.sh

```diff
@@ -1,79 +1,54 @@
 #!/usr/bin/env bash
-#
-# Bring up the SDK test environment.
 
-set -e
+set -ae
```

Comment:
@tzaffi Optional:   Since I reviewed the shell configuration for exports, I optionally _suggest_ the configuration I default to (as discussed in https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/).

* Treat optionally - I don't feel strongly.  I leave the suggestion because the configuration has helped me track down errors.
* I deliberately omit `-x`.  I don't think it's needed by default.

```suggestion
set -euo pipefail
```

Reply:
will try